mautic/mautic
# Add log processor

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ❌ 
| New feature/enhancement? (use the a.x branch)      | ✅ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | todo  
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



We need hostname and pid with logs that are being created as an extra info for log, this will help us debug issues through logs faster. This PR add log processor.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Run segment or campaign update or campaign trigger etc commands
3. Check logs for hostname and pid in it




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11121"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
